prometheus/README.md

```diff
@@ -3,48 +3,18 @@
 The Prometheus Event Source enables Knative Eventing integration with the
 [Prometheus Monitoring System](https://prometheus.io/).
 
-At the moment the implementation supports only simple PromQL queries. The
-queries run against the Prometheus server every 5 seconds and the results are
-sent to the configured sink as CloudEvents.
+## Instant and Range Queries
 
-## Deployment steps
-
-### Kubernetes
```

Comment:
I was still liking that as an example to external source - let's keep it

Reply:
Added a section on off-cluster Prometheus server, please see the next commit.